src/styles/blocks/card.scss

```diff
@@ -0,0 +1,96 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 200px;
+  padding: 32px 16px 16px;
+  font-family: "Roboto", sans-serif;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: #060b35;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: #616070;
+    width: 100%;
+    margin-bottom: 16px;
+  }
+
+  &__stars-wrapper {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 100%;
+    margin-bottom: 24px;
+  }
+
+  &__feedback {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: #060b35;
+  }
+
+  &__price-wrapper {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 100%;
+    margin-bottom: 16px;
+  }
+
+  &__price-title {
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 18px;
+    color: #616070;
+  }
+
+  &__price {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: #060b35;
+  }
```

Comment:
```suggestion
  &__price {
    font-weight: 700;
    font-size: 16px;
    line-height: 18px;
    color: #060b35;

    &-title {
      font-weight: 400;
      font-size: 12px;
      line-height: 18px;
      color: #616070;
    }

    &-wrapper {
      display: flex;
      justify-content: space-between;
      align-items: center;
      width: 100%;
      margin-bottom: 16px;
    }
  }
```

Reply:
`&__price-wrapper`, ` &__price-title` and ` &__price` are different part of block `card` and nesting does not fit here

Reply:
better to rename `&__price` to `&__price-velue`

Reply:
it's up to you, I also though that was weird in this case, but I heard that if I can nest something, I should do it